ERCS/erc-xxx.md

```diff
@@ -0,0 +1,441 @@
+---
+title: Structured Data Clear Signing Format
+description: Draft specification of a json format providing additional
+  description of how to clear-sign smartcontract calls and EIP712 messages on
+  any wallets
+author: Laurent Castillo(@lcastillo-ledger)
+discussions-to: https://github.com/LedgerHQ/clear-signing-format-spec/issues
```

Comment:
Please create a discussions link in the Eth Magicians forum linking to this PR.  https://ethereum-magicians.org/

Reply:
Hi Andrew,

I'm trying to but my github account being too new on ethereum magicians, I can't post new topics. Can you help me post?

Thanks!